packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -632,33 +642,34 @@ export default {
 
             try {
                 await this.submitCheckout();
+                this.trackFormInteraction({ action: 'success' });
             } catch (error) {
                 this.handleErrorState(error);
             } finally {
                 this.shouldDisableCheckoutButton = false;
             }
         },
 
+        /**
+         * Check to see if any `Vuelidate` validation errors
```

Comment:
Doesn't need to be part of this PR, but could we update these JSDoc comments to include a `@returns {type}` at the bottom of them? Just helps when debugging down the line.

[Like this sort of thing in searchbox](https://github.com/justeat/fozzie-components/blob/master/packages/components/molecules/f-searchbox/src/components/Form.vue#L201-L206).

Reply:
I'll do this as a separate PR. This one is getting a bit big 😄